.gitlab-ci.yml

```diff
@@ -171,3 +173,30 @@ force_release_prod:
     RELEASE_COMMAND: scripts/release.sh staging prod --force
   only:
     - staging
+
+verify_sync_partial:on-schedule:
```

Comment:
There is a lot of duplicate code between `verify_sync_partial:on-schedule` and `verify_sync_full:on-schedule:` can you consolidate?

Reply:
Good call, done!